tests/code_context_test.py

```diff
@@ -204,22 +204,22 @@ async def test_get_code_message_cache(
         "mentat.code_context.CodeContext._get_code_message"
     )
     mock_get_code_message.return_value = "test1"
-    value1 = await code_context.get_code_message("gpt-4", 1e6)
+    value1 = await code_context.get_code_message("", "gpt-4", 1e6)
```

Comment:
no need to change, but it'd be more clear if we called with keyword args, i.e. `get_code_message(prompt="", model="gpt-4", ...)